stylesheet.css

```diff
@@ -1,146 +1,155 @@
-body{
-    background-color: #161316;
-}
-
-.mainBannerContainer{
-    position: auto;
-    display: block;
-    width: 1300px;
-    height: 780px;
-    margin-top: 20px;
-    margin-left:auto;
-    margin-right:auto;
-    margin-bottom: 0;
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
 }
 
-.mainBodyContainer{
-    position: auto;
-    display: block;
-    width: 1300px;
-    height: auto;
-    margin-top: 0px;
-    margin-left:auto;
-    margin-right:auto;
-    margin-bottom: 0;
-    /* border-style: solid; 
-    border-color: aliceblue; */
+:root {
+  --grey: #c6c5c5;
+  --white: #ffffff;
+  --header: #2da4af;
+}
+
+body {
+  margin: 0 auto;
+  background-color: #161316;
+  max-width: 1300px;
+}
```

Comment:
Let's determine our max-width here, then the rest of our elements can just grow to 100%.